### PR TITLE
Create properties plugin

### PR DESCRIPTION
**Pull request by janstenpickle**
Waiting on @nathankleyn to finish changes to shanty core before this can be merged

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,125 @@
+PATH
```
Comment (nathankleyn) [inline Gemfile.lock]: Should be removed and added to `.gitignore`.

File: LICENCE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 
+Copyright (c) 2015 Shanty
```
Comment (nathankleyn) [inline LICENCE]: Best to put our names here for now, as Shanty isn't a company or anything we can prove ownership of.

File: lib/shanty_properties_file_plugin.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Public: shanty properties file plugin module
```
Comment (nathankleyn) [inline lib/shanty_properties_file_plugin.rb:4]: This file should require the other files in your gem that you want to load when somebody does `require 'shanty_properties_file_plugin'`.

Comment (janstenpickle) [inline lib/shanty_properties_file_plugin.rb:4]: Fixed

**Pull request merged**